### ISSUE TITLE
number of training images for imagenet

### ISSUE THREAD

**Issue by xulj-cs** [thuml/HashNet#43]
Hi, first really appreciate for sharing codes. I have a question about the experimental settings.
in your paper, experimental settings for imagenet is

> We randomly select 100 categories, use all the images of these categories in the training set as the database, and use all the images in the validation set as the queries; furthermore, we randomly select 100 images per category from the database as the training points

where the total number of training images is 10,000. However, the number of training images in this repo is 13,000.
Which one is right to reproduce your results in the paper?

**Comment by yuleung** [comment]
Same question.
